expected/init_test_healpix.out.in

```diff
@@ -1,2 +1,3 @@
 psql:pg_sphere.test.sql:9207: NOTICE:  return type smoc is only a shell
 psql:pg_sphere.test.sql:9213: NOTICE:  argument type smoc is only a shell
+
```

Comment:
Remove this blank line. In this file, change "9207" to "9221" and change "9213" to "9227".